docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Understanding Lit PKPs as escrow
+
+A Private key is meant for signing transactions which are executed on the blockchain, a network verifies if the transaction is a legitimate one and then executes it. With Lit Network we can generate a special type of private keys which can be programmed to sign transactions only when certain conditions are met. With this, we can create several escrow mechanisms, and we will look into how we can execute a cross-chain swap.
+
+### A swap example would be:
+Suppose Alice holds 100 token A on Base and Bob holds 100 token B on Chronicles and they both agree to a swap of,
```

Comment:
here can we just reference the actual tokens used in the example code? Like, "in the example below, Alice holds x tokens on Base and Bob holds x tokens on Chronicle. They agree to the following swap parameters:"

Reply:
I wrote this on testnets and so using my own custom-minted tokens. 
I designed it in a way if anyone wants to use the example as well, they can mint these tokens for themselves.

I can replace these with the token addresses (but that would make it too long, it's used A and B at many places), or for this example, we can assume eth and dai from the start and add a little disclaimer around conditions section where addresses are used that these are our representation of eth and dai on testnets.

Reply:
like i've added more context on revamping the below statement

Below are the parameters for the cross-chain swap. In the example below, we'll be transferring 4 tokens A on Sepolia for 8 tokens B on Chronicle Yellowstone. These tokens are minted by a custom contract.

Reply:
cool, this works